src/Apitizer/Types/Filter.php

```diff
@@ -23,6 +23,11 @@ class Filter extends Factory
      */
     protected $format = null;
 
+    /**
+     * @var array<string> the format for date(time) types.
+     */
+    protected $enums = null;
+
```

Comment:
The documentation seems to be from the `format`. Could you update this to correctly document enums?

Reply:
Changed to correct documentation